WooCommerce/Classes/ViewRelated/Dashboard/Coupons/MostActiveCouponsCard.swift

```diff
@@ -110,6 +124,17 @@ private extension MostActiveCouponsCard {
         }
     }
 
+    var emptyView: some View {
+        HStack {
+            Spacer()
+
+            MostActiveCouponsEmptyView()
+
+            Spacer()
+        }
```

Comment:
Just a suggestion: you can avoid adding `Spacer()` by setting the width of the view to the maximum - that should also push it to the center.

```suggestion
        MostActiveCouponsEmptyView()
            .frame(maxWidth: .infinity)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12755/commits/5079a5ddf9ad1c8fd39245159d29119a55a6b1d7